core/server/models/base/index.js

```diff
@@ -114,7 +114,7 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
             newObj.setId();
         }
 
-        if (!this.get('created_by')) {
+        if (schema.tables[this.tableName].hasOwnProperty('created_by') && !this.get('created_by')) {
```

Comment:
Ok that's a good idea. Let me add a comment that all models need to ensure calling the base events 👍 
And i will explain why.